packages/react/modules/hooks/use-adapter-subscription/use-adapter-subscription.ts

```diff
@@ -1,20 +1,34 @@
-import { TAdapter } from '@flopflip/types';
+import { TAdapter, TAdapterSubscriptionStatus } from '@flopflip/types';
 import React from 'react';
 
 function useAdapterSubscription(adapter: TAdapter) {
+  const useAdapterSubscriptionStatusRef = React.useRef(
+    TAdapterSubscriptionStatus.Subscribed
+  );
```

Comment:
This is the trick. For know we duplicate this state in the react component. Making it easier for consumers cause nothing changes.

Reply:
Ah nice solution!